recipes/canu/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: canu
+  version: 1.1
+
+source:
+  # Canu currently needs to come from git to get it's version string right
+  git_url: https://github.com/marbl/canu.git
```

Comment:
If not running in a git checkout Canu does not set it's version information. We could hack out the canu_version_update.pl step and manually create canu_version.H if that's preferable (seemed ugly to me).